tests/redis_valgrind.sup

```diff
@@ -15,3 +15,203 @@
    Memcheck:Value8
    fun:lzf_compress
 }
+
+{
+   <redis_dumpCommand_supression>
+   Memcheck:Value8
+   fun:crcspeed64little
+   fun:crcspeed64native
+   fun:crc64
+   fun:createDumpPayload
+   fun:dumpCommand
+}
+
+
+{
+   <redis_rioGenericUpdateChecksum_supression>
+   Memcheck:Value8
+   fun:crcspeed64little
+   fun:crcspeed64native
+   fun:crc64
+   fun:rioGenericUpdateChecksum
+   fun:rioWrite
+}
+
+
+{
+   <redis_dumpCommand_supression>
+   Memcheck:Value8
+   fun:crcspeed64little
+   fun:createDumpPayload
+   fun:dumpCommand
+}
+
+{
+   <uninitialised_bytes_connWrite>
+   Memcheck:Param
+   write(buf)
+   fun:__libc_write
+   fun:write
+   fun:connSocketWrite
+   fun:connWrite
+}
+
+{
+   <invalid_write_size_8_compression_appendFloat>
+   Memcheck:Addr8
+   fun:appendBits
+   fun:appendFloat
+   fun:Compressed_Append
+   fun:Compressed_AddSample
+}
+
+{
+   <invalid_write_size_8_compression_appendInteger>
+   Memcheck:Addr8
+   fun:appendBits
+   fun:appendInteger
+   fun:Compressed_Append
+   fun:Compressed_AddSample
+}
+
+{
+   <temporary_supression_redistimeseries_UNCOMPRESSED_due_to_untrimmed_series_DUMP_LOAD>
```

Comment:
@danni-m @gkorland 
temporarily includes supression of Series DUMP/RESTORE due to untrimmed series with empty chunks being stored as is ( meaning we can have chunks that are empty saved ).

Reply:
see #609